### PR TITLE
raidboss: initial triggers for The Masked Carnivale

### PR DESCRIPTION
**Pull request by xiashtra**
## Description

Adds triggers for The Masked Carnivale Stages 01-31.

## Additional Notes
All current Masked Carnivale stages take place within the same zone.  There are separate InstanceContentIDs for each Stage, which can be seen in Actor Control lines.  If necessary, these could be used to uniquely identify which Stage has been loaded.

The ContentTypeID for The Masked Carnivale is missing from `content_type.ts` (re: #5061).

@quisquous I wasn't sure where in the content hierarchy this content should exist.  It was first added in Stormblood (4.5), and further expanded in Shadowbringers (5.15, 5.45).  However, for a new player, the content is accessible as early as finishing ARR MSQ (2.0), with additional Stages unlocked after Heavensward MSQ (3.3) and Stormblood MSQ (4.3).  Should there be another category for content that lives outside the standard Duty Finder?  It could possibly incorporate things like The Masked Carnivale, Eureka, Bozja, Variant & Criterion dungeons, deep dungeons, etc.

For now I placed this under Stormblood dungeons, but that doesn't seem like a good place for it.

### CODE REVIEW

**Review by quisquous — COMMENTED** [review]
These look good in general.  Do you have any desire to add the oopsy file for this too for completeness? If not, do you have logs for all of these and I can do it?

**Comment by xiashtra** [comment]
> These look good in general. Do you have any desire to add the oopsy file for this too for completeness? If not, do you have logs for all of these and I can do it?

I haven't tried doing anything with Oopsy yet, if it's fairly straightforward I could probably get to it in a separate pull request next week.

File: ui/raidboss/data/04-sb/dungeon/the_masked_carnivale.ts [hunk 1]
```diff
@@ -0,0 +1,1232 @@
...
+    },
+    {
+      id: 'Carnivale S26 A2 Papa Humbaba Void Thunder III',
+      // 10F = Electrocution, can be cleansed
+      type: 'GainsEffect',
+      netRegex: { effectId: '10F', source: 'Papa Humbaba' },
+      infoText: (data, matches, output) => output.text!({ player: data.ShortName(matches.target) }),
+      outputStrings: {
+        text: {
+          en: 'Cleanse ${player}',
```
[the remark this answers is not in the document]
Comment (xiashtra) [reply ui/raidboss/data/04-sb/dungeon/the_masked_carnivale.ts]: Simplified all of the cleanse callouts to assume solo player.

**Comment by quisquous** [comment]
> I haven't tried doing anything with Oopsy yet, if it's fairly straightforward I could probably get to it in a separate pull request next week.

Yeah, it's fairly straightforward in general.  Mostly just like, what are the ability ids that you shouldn't be hit by and what are the effects that you shouldn't get.

I keep meaning to write some sort of log filterer that can pull these out more clearly, but my hacky workflow is to import logs into ACT and use this organically-grown regex when looking at fights: `( 257 | 1B:| 1[456]:4.......:(?!\w*-Egi|Liturgic Bell|Thancred|Moonstone Carbuncle|Alphinaud|Alisaie|Topaz Titan|Emerald Carbuncle|Eos|Rook Autoturret|Seraph|Bishop Autoturret|Ruby Carbuncle|Esteem|Demi-\w*|Earthly Star|Automaton Queen|Bunshin|Selene|Hien|Lyse|Pipin Of The Steel Heart|Yugiri Mistwalker|2B|Topaz Carbuncle|Y'shtola|Hythlodaeus|Emet-Selch|Venat|Krile|Estinien|Urianger|G'raha Tia|Carbuncle|Varshahn|Emerald Garuda|Ruby Ifrit)[^:]*:(?!5B1A|5CEC|5CED|5CEE|5CEF|6143|5B1A|53F2|41BE|5462|6F73|6F74)[^:]*:(?!Attack|attack))`

**Pull request merged**